earth_enterprise/src/server/mod_fdb/SConscript

```diff
@@ -21,21 +22,27 @@ import os
 env = env.Clone();
 env.Append(CPPPATH = ['#common/geindex'])
 
-module = env.sharedLib('mod_fdb',
+mod_fdb_libs = ['glb', 'qtutils', 'serverdb', 'serverdb_r', 'cutspec', 'gdal']
+
+fdb_module = env.sharedLib('mod_fdb',
                ['mod_fdb.c', 'mod_fdb_impl.cpp', 'apache_fdb_reader.cpp',
                 'unpackermanager.cpp', 'fdb_reader_manager.cpp',
                 'fdbservice.cpp', 'portableservice.cpp', 'fusiondbservice.cpp',
                 'tileservice.cpp', 'motf_generator.cpp'],
-               LIBS=['glb', 'qtutils', 'serverdb',
-                     'serverdb_r', 'cutspec', 'gdal'],
-               LINK=['g++'])
+               LIBS=mod_fdb_libs,
+               LINK=['g++'],
+               # Make sure scons gets the soname right since it isn't named
+               # like a normal library.
+               LINKFLAGS=env['LINKFLAGS'] + ['-Wl,-soname,mod_fdb.so'])
+
 
 # Install /opt/google/gehttpd/auth directory only
 auth_dir = env.installdirs['httpd'].entry_abspath('auth')
 if (not os.path.exists(auth_dir)):
     os.makedirs(auth_dir)
 
-env.install('httpdmodules', [module])
+#env.install('server_lib', [fdb_module])
```

Comment:
should we delete the line instead of commenting?

Reply:
Yes, good catch